src/site/navigation/facility_sidebar_nav.drupal.liquid

```diff
@@ -1,4 +1,4 @@
-<nav id="va-detailpage-sidebar" data-drupal-sidebar class="va-c-facility-sidebar usa-width-one-fourth va-sidebarnav">
+<nav id="va-detailpage-sidebar" class="va-c-facility-sidebar usa-width-one-fourth va-sidebarnav">
```

Comment:
I think the JS for triggering the sidebar still depends on `data-drupal-sidebar`